src/components/List/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { memo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import ListItem from 'components/ListItem';
+import { Articles } from 'types/Article';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  data: Articles | undefined;
+}
+
+function List({ data }: Props) {
+  const { t } = useTranslation('List');
+  return (
+    <div>
+      {data?.articles && data.articles.map((article) => <ListItem article={article} key={article.slug} />)}
+      {data?.articlesCount === 0 && <div className={styles.emptyMessage}>{t('empty')}</div>}
```

Comment:
This is valid, but there's a cleaner way, you could simply precede the condition with a `!` sign.
```
!data?articlesCount && <div...
```